test/grouping.jl

```diff
@@ -146,8 +146,9 @@ module TestGrouping
     groupby(dt, [:v1, :v2])
 
     dt2 = by(e->1, DataTable(x=Int64[]), :x)
-    @test size(dt2) == (0,1)
+    @test size(dt2) == (0,2)
     @test isequal(sum(dt2[:x]), Nullable(0))
+    @test size(by(e->1, DataTable(x=Int64[1]), :x)) == (1,2)
```

Comment:
Why add this line in particular, and check only its size? It looks out of place in this block. Either remove it or make it more complete in its own block.